app/controllers/api/schedules_controller.rb

```diff
@@ -14,26 +16,31 @@ def create
     #   render json: schedule
     # end
 
-    # def update
-    #   mutate Schedules::Update.run(params[:schedule],
-    #                                user: current_user)
-    # end
+    def update
+      if schedule.user != current_user
+        raise Errors::Forbidden, "Not your schedule."
+      end
+      mutate Schedules::Update.run(params[:schedule],
+                                   user: current_user,
+                                   schedule: schedule)
+    end
 
-    # def destroy
-    #   if (schedule.user == current_user) && schedule.destroy
-    #     render nothing: true
-    #   else
-    #     raise Errors::Forbidden, "Not your schedule."
-    #   end
-    # end
+    def destroy
+      if (schedule.user == current_user) && schedule.destroy
+        render nothing: true
+      else
+        raise Errors::Forbidden, "Not your schedule."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.